torchdynamo/output_graph.py

```diff
@@ -433,3 +433,25 @@ def cleanup(self):
         for node in self.graph.nodes:
             if "example_value" in node.meta:
                 del node.meta["example_value"]
+
+    def create_proxy(
+        self,
+        kind,
+        target,
+        args,
+        kwargs,
+        name=None,
+        type_expr=None,
+        proxy_factory_fn=None,
+        current_tx=None,
```

Comment:
Soft suggestion - feel free to ignore - replace `current_tx` to just `tx`?

Reply:
I will keep it as current_tx, as it's a stack of txs....